apps/api/src/config/database/configuration.ts

```diff
@@ -15,6 +15,7 @@ export class DatabaseConfiguration implements TypeOrmOptionsFactory {
       username: this.configService.getOrThrow<string>('DB_USERNAME'),
       password: this.configService.getOrThrow<string>('DB_PASSWORD'),
       database: this.configService.getOrThrow<string>('DB_NAME'),
+      logging: this.configService.getOrThrow<boolean>('LOGGING'),
```

Comment:
This is not needed remove this.. Also its incorrect like I mentioned earlier.

Reply:
Removed